docs/formats.rst

```diff
@@ -648,20 +648,20 @@ JSON i18next files
 
 .. versionchanged:: 2.17
 
-    Since Weblate 2.17 and with translate-toolkit at least 2.2.5 i18next
+    Since Weblate 2.17 and with translate-toolkit at-least 2.2.5, i18next
     JSON files with plurals are supported as well.
 
-`i18next <https://www.i18next.com/>`_ is an internationalization-framework
+`i18next <https://www.i18next.com/>`_ is an internationalization framework
 written in and for JavaScript. Weblate supports its localization files with
 features such as plurals.
 
-i18next translations are monolingual, so it is recommended to specify base file
-with English strings.
+i18next translations are monolingual, so it is recommended to specify a base file
+with (what is most often the) English strings.
 
 .. note::
 
-   Weblate supports i18next JSON v3 format. The v2 and v1 variants are mostly
-   compatible, with exception of handling plurals.
+   Weblate supports the i18next JSON v3 format. The v2 and v1 variants are mostly
+   compatible, with the exception of handling plurals.
```

Comment:
exception of not handling ### ~they are not handled
OR
exception of how plurals are handled ### ~they are handled differently

Reply:
They are handled differently, see https://www.i18next.com/misc/json-format#i-18-next-json-v3:

> The only difference to v2 are the plural suffixes for languages with multiple plural forms.

Reply:
already fixed this few days ago and forgot to resovle this convesation. Sorry.